scilpy/utils/metrics_tools.py

```diff
@@ -212,3 +212,30 @@ def plot_metrics_stats(mean, std, title=None, xlabel=None,
 
     plt.close(fig)
     return fig
+
+
+def get_metrics_stats_over_volume(weights, metrics_files):
+    """
+    Returns the mean and standard deviation of each metric, using the
+    weighting matrix provided in weights. This can be a binary mask,
+    or contain weighting factors between 0 and 1.
+
+    Parameters
+    ------------
+    data : 3d-array (X, Y, Z)
+        3D numpy array containing a weighting matrix
+
+    metrics_files : sequence
+        list of nibabel objects representing the metrics files
+
+    Returns
+    ---------
+    stats : list
+        list of tuples where the first element of the tuple is the mean
+        of a metric, and the second element is the standard deviation
+
+    """
+
+    return map(lambda metric_file:
+               weighted_mean_stddev(weights,metric_file.get_fdata(np.float64)),
```

Comment:
stddev -> std_dev ?
Also should we used float32?

Reply:
I really don't know it was cast as np.float64 in the private repository.
I don't feel like it will make a huge difference and should be cast as float32...

@jchoude, @GuillaumeTh  ?

Reply:
change for a cast as float32 same for L71.

Reply:
I was casting to f64 in the old version because, as some point, we had issues with metrics that have a small range (AD, MD and such), where the `stddev` was not stable when using f32 for computations.